pkg/network/config/config.go

```diff
@@ -201,9 +204,10 @@ func New() *Config {
 		MaxDNSStatsBuffered: 75000,
 		DNSTimeout:          time.Duration(cfg.GetInt(join(spNS, "dns_timeout_in_s"))) * time.Second,
 
-		EnableHTTPMonitoring:  cfg.GetBool(join(netNS, "enable_http_monitoring")),
-		EnableHTTPSMonitoring: cfg.GetBool(join(netNS, "enable_https_monitoring")),
-		MaxHTTPStatsBuffered:  100000,
+		EnableHTTPMonitoring:            cfg.GetBool(join(netNS, "enable_http_monitoring")),
+		EnableHTTPSMonitoring:           cfg.GetBool(join(netNS, "enable_https_monitoring")),
+		EnableHTTPHTTPSMonitoringViaETW: cfg.GetBool(join(netNS, "enable_http_https_monitoring_via_etw")),
```

Comment:
I think if `EnableHTTPHTTPSMonitoringViaETW` is enabled, that should imply that http monitoring is enabled, so we should set those configuration values accordingly